src/texstudio.cpp

```diff
@@ -1403,9 +1403,8 @@ void Texstudio::setupMenus()
         }
     }
 
-	newManagedAction(menu, "enlargePDF", tr("Show embedded PDF large"), SLOT(enlargeEmbeddedPDFViewer()));
-	newManagedAction(menu, "shrinkPDF", tr("Show embedded PDF small"), SLOT(shrinkEmbeddedPDFViewer()));
-
+	newManagedAction(menu, "enlargePDF", tr("Show embedded PDF large"), SLOT(enlargeEmbeddedPDFViewer()))->setVisible(false);
```

Comment:
please make two lines out of this, i.e 
act=...
act->setVi...

Reply:
and the approach seems to be a bit indirect.
You want to change menu entries in texstudio.cpp level, the changes are triggered there (even embedding/windowing over Texstudio::runInternalPdfViewer)